package.json

```diff
@@ -69,6 +69,7 @@
     "less": "^3.12.0",
     "less-middleware": "^3.0.1",
     "lodash": "^4.17.19",
+    "lodash.isequal": "^4.5.0",
```

Comment:
Do we have to do this still? It's deprecated and unsupported upstream. Can we try named imports from `lodash`? Tree shaking should be a thing maybe.